tardis/model/tests/test_csvy_model.py

```diff
@@ -42,16 +42,48 @@ def test_compare_models(filename):
                 config_model_val = config_model_val.value
                 csvy_model_val = csvy_model_val.value
             npt.assert_array_almost_equal(csvy_model_val, config_model_val)
+    pd.testing.assert_frame_equal(csvy_model.abundance,config_model.abundance,check_less_precise = True)
+    pd.testing.assert_frame_equal(csvy_model.raw_abundance,config_model.raw_abundance)
+    pd.testing.assert_frame_equal(csvy_model.raw_isotope_abundance,config_model.raw_isotope_abundance)
 
-
-def test_csvy_abundance():
-    csvypath = os.path.join(DATA_PATH, 'config_v_filter.yml')
+@pytest.fixture(scope='module')
+def csvy_hard_coded():
+    csvypath = os.path.join(DATA_PATH, 'csvy_hard_coded.yml')
     config = Configuration.from_yaml(csvypath)
-    csvy_model = Radial1DModel.from_csvy(config)
-    csvy_abund = csvy_model.abundance
+    csvy_hard_coded = Radial1DModel.from_csvy(config)
+    return csvy_hard_coded 
+ 
+def test_csvy_input_abundance_storage(csvy_hard_coded):
+
+    #test abundance
+    abundance_index = pd.Index([1,2],name = 'atomic_number')
+    input_abundance = pd.DataFrame([[0.0,0.33,0.3,0.5,0.4,0.2], 
+                                    [0.98,0.64,0.6,0.4,0.55,0.79]],
+                                   index = abundance_index)
+
+    pd.testing.assert_frame_equal(csvy_hard_coded.raw_abundance,input_abundance,check_names = True)
+    
+    #test isotopes
+    arrays = [[28],[56]]
+    isotope_index = pd.MultiIndex.from_arrays(
+            arrays, names=["atomic_number", "mass_number"]
+        )
+    input_isotopes = pd.DataFrame([[0.02,0.03,0.1,0.1,0.05,0.01]],
+                                  columns=np.arange(6),
+                                  index = isotope_index)
+    
+    pd.testing.assert_frame_equal(csvy_hard_coded.raw_isotope_abundance,input_isotopes,check_names = True)
+    
 
-    ref_abund = pd.DataFrame(np.array([[0.35,0.3, 0.6, 0.4],[0.65,0.7,0.4,0.6]]))
-    ref_abund.index.name = 'atomic_number'
-    ref_abund.index = np.array([1, 2])
+#test decay
+def test_decay_dataframe(csvy_hard_coded):
+    decay_index = pd.Index([1, 2, 26, 27, 28],name = 'atomic_number')
+    ref_abund = pd.DataFrame([
+                      [0.0,0.33,0.3,0.5,0.4,0.2], 
+                      [0.98,0.64,0.6,0.4,0.55,0.79],
+                      [0.00013977843354947162, 0.00020966765032420787, 0.0006988921677473642, 0.0006988921677473642, 0.0003494460838736821, 6.988921677473581e-05],
+                      [0.007188928223953217, 0.010783392335929825, 0.035944641119766085, 0.035944641119766085, 0.017972320559883043, 0.0035944641119766084],
+                      [0.012671293342497312, 0.019006940013745966, 0.06335646671248656, 0.06335646671248656, 0.03167823335624328, 0.006335646671248656]
```

Comment:
Why are these hardcoded numbers different from the hardcoded numbers for the config_decay test?  Let's just use one set of hardcoded numbers.

Reply:
Why do we have two separate test files for the config and csvy models?